WebService/v1/Controllers/StatusController.cs

```diff
@@ -70,15 +70,15 @@ public sealed class StatusController : Controller
         }
 
         // TODO: reduce method complexity, refactor out some logic
-        [HttpGet]
-        public async Task<StatusApiModel> GetAsync()
+        [HttpGet("{id}")]
+        public async Task<StatusApiModel> GetAsync(string id)
```

Comment:
I think we should drop simulation data from the /status endpoint. Indeed I suggested Hugh to move simulation stats from here to GET /simulation/<id>

Unless you already hooked up the UI to pass the ID, I would remove this.  Only useful info we could leave here is the number of running simulations, in future.

By the way, how will the UI know the simulation ID? I assume it will get all and use the first from the list? we might want to discuss this a bit